ts/Series/Wordcloud/WordcloudPoint.ts

```diff
@@ -0,0 +1,61 @@
+/* *
+ *
+ *  Experimental Highcharts module which enables visualization of a word cloud.
+ *
+ *  (c) 2016-2020 Highsoft AS
+ *  Authors: Jon Arild Nygard
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ * */
+
+
+import type SizeObject from '../../Core/Renderer/SizeObject';
+import type WordcloudPointOptions from './WordcloudPointOptions';
+import ColumnSeries from '../Column/ColumnSeries.js';
```

Comment:
Because world cloud is its own module, you have to import series dependencies over `BaseSeries`, to not increase the product JS file size.